src/contexts/Balance/index.ts

```diff
@@ -0,0 +1,2 @@
+export { BalanceContext } from './Context';
+export { BalanceProvider } from './Provider';
```

Comment:
It's honestly kinda weird to me that this is split on 3 files... Feels a bit bloaty—having declarations and exports on one file is not bad.

Reply:
Fixed in a0f4757d01ce1b4ff2b283fd8e81c6bacc09054f